src/runtime_src/tools/xclbinutil/XclBinUtilities.cxx

```diff
@@ -869,6 +869,14 @@ XclBinUtilities::createMemoryBankGrouping(XclBin & xclbin)
       // Transform and group the memories
       transformMemoryBankGroupingCollections(connectivity, groupTopology, groupConnectivity);
 
+      // Perform some DRC checks on the memory grouping and connectivity produced before marging to group connectivity
+      validateMemoryBankGroupEntries((unsigned int) memTopology.size(), groupTopology, groupConnectivity);
```

Comment:
The reason why this validation was done at the end was to validate the contents and connections of the two GROUP sections.  Now that is no longer the case.

Why couldn't the validation method be updated?